src/page/input-demo.js

```diff
@@ -119,6 +124,63 @@ export class InputDemo extends React.Component {
             </Panel>
           </Col>
           <Col>
+            <Panel title='Editable Select Dropdowns'>
+              <Col>
```

Comment:
It looks like we have `<Col>` inside a `<Col>`. We want to make sure anything we need nested `<Col>`s that we put the nested ones inside a `<Row>`